benchmarks/source-agilitycms/README.md

```diff
@@ -0,0 +1,31 @@
+<p align="center">
+  <a href="https://gatsbyjs.org">
+    <img alt="Gatsby" src="https://www.gatsbyjs.org/monogram.svg" height="60" />
+    <img alt="Gatsby" src="https://static.agilitycms.com/layout/img/logo-original.svg" width="200" />
+  </a>
+</p>
+<h1 align="center">
+  Gatsby + Agility CMS : WILL IT BUILD
+</h1>
+
+## To Update Content from the CMS
+
+```shell
+num run update
+```
+
+## Pull Content From Agility CMS
+
+For the first time setup only (this has been done by Agility CMS staff)
+
+```shell
+npm run setup
+```
+
```

Comment:
While a little overkill, I think it'd be good to be explicit in how to build, even if it's "just the default". This prevents an engineer potentially overthinking it, looking up to make sure that it needs no other args.

````suggestion
## Build

```shell
gatsby build
```
````